openstack_ansibleee/Dockerfile

```diff
@@ -18,10 +18,15 @@ COPY --from=builder /usr/share/ansible /usr/share/ansible
 
 COPY $REMOTE_SOURCE/openstack_ansibleee/settings /runner/env/settings
 RUN chmod 775 /runner/env/settings
-COPY $REMOTE_SOURCE/openstack_ansibleee/edpm_entrypoint.sh /bin/edpm_entrypoint
-RUN sed -i '1d' /bin/entrypoint
-RUN cat /bin/entrypoint >> /bin/edpm_entrypoint
-RUN chmod +x /bin/edpm_entrypoint
+COPY $REMOTE_SOURCE/openstack_ansibleee/edpm_entrypoint.sh /opt/builder/bin/edpm_entrypoint
+RUN sed -i '1d' /opt/builder/bin/entrypoint
+RUN cat /opt/builder/bin/entrypoint >> /opt/builder/bin/edpm_entrypoint
+RUN chmod +x /opt/builder/bin/edpm_entrypoint
 ENV EDPM_SYSTEMROLES='fedora.linux_system_roles'
+RUN chmod ug+rw /etc/passwd
+RUN mkdir -p /runner && chgrp 0 /runner && chmod -R ug+rwx /runner
 WORKDIR /runner
-ENTRYPOINT ["edpm_entrypoint"]
+LABEL ansible-execution-environment=true
+USER 1000
+ENTRYPOINT ["/opt/builder/bin/edpm_entrypoint", "dumb-init"]
+CMD ["bash"]
```

Comment:
is the CMD needed?

Reply:
I just tried to follow the ansible-builder output: https://gist.github.com/fao89/64b2f35d6d5edfb8c96afe525c817c7e

Reply:
but I don't think it is needed

Reply:
with an empty execution-environment.yaml file, I get different results than that gist. The base image used is actually, quay.io/ansible/ansible-runner:latest.

I wonder if we could that as our base image instead of ghcr.io/ansible/community-ansible-dev-tools:latest

Reply:
that image seems to be 2 years old:https://quay.io/repository/ansible/ansible-runner?tab=tags&tag=latest
my execution-environment.yaml was based on https://ansible.readthedocs.io/projects/dev-tools/container/#usage  (without the python requirements and package manager)

Reply:
I started this: https://github.com/ansible/ansible-builder/pull/715 so we can see what ansible-builder people have to say about the image

Reply:
From https://github.com/ansible/ansible-builder/pull/715#issuecomment-2427624093 it seems quay.io/ansible/ansible-runner is there to keep compatibility with older versions

Reply:
ok, then I think we can proceed with this.
In the longer term, we might want to look at migrating the image build to use ansible-builder directly instead of maintaining our own method with the Containerfile.

Reply:
I filed https://issues.redhat.com/browse/OSPRH-11080 to track that work

Reply:
we can follow ansible-dev-tools image example, as it is built with ansible-builder: https://github.com/ansible/ansible-dev-tools/blob/main/execution-environment.yml